react/components/SelectionListItem.js

```diff
@@ -21,6 +21,7 @@ const SelectionListItem = ({ option, onItemClick, selected }) => {
       className={classNames(
         handles.orderByOptionItem,
         highlight,
+        selected && handles.orderByOptionSelected,
```

Comment:
I think the `applyModifiers` is the right way, you just have to pass an empty string if `selected` is `false`

```suggestion
        applyModifiers(handles.orderByOptionItem, selected ? 'selected : ''),
```